pix2pix/pix2pix_website/README.md

```diff
@@ -0,0 +1,57 @@
+# Pix 2 Pix demo
+
+A demonstration of Deep Learning and FaaS
+
+## Installation
+From the pix2pix_website directory, install the required npm packages.
+```
+npm install
+```
+# Usage
```

Comment:
Can we add a "pre requisites" section specifying you need an openfaas deployment?

Reply:
Sound good, added.